roboflow/core/workspace.py

```diff
@@ -273,9 +271,9 @@ def upload_dataset(
         dataset_path: str,
         project_name: str,
         num_workers: int = 10,
-        dataset_format: str = "yolov8",
```

Comment:
this is always automatically detected now?

One thif about this vhnage is that it changes the function signature or user facing function.  Should we keep it and print a deprecation warning instead if parsed?

Not a problem if people are using named parameters, but if they have a script that passes the raw values in order, this might break existing scripts?

Reply:
Great point. Lemme fix that.